include/xsimd/arch/xsimd_avx512f.hpp

```diff
@@ -1193,6 +1193,38 @@ namespace xsimd
             return detail::compare_int_avx512f<A, T, _MM_CMPINT_LE>(self, other);
         }
 
+        namespace detail
+        {
+            // Generate a bitset from an array of boolean.
+            XSIMD_INLINE unsigned char tobitset(unsigned char unpacked[8])
+            {
+                uint64_t data;
+                memcpy(&data, unpacked, sizeof(uint64_t));
+
```

Comment:
can you quote  https://github.com/serge-sans-paille/fast-bitset-from-bool-array here?